modules/face/src/facemarkLBF.cpp

```diff
@@ -1212,8 +1212,9 @@ Mat FacemarkLBFImpl::Regressor::supportVectorRegression(
     double Gnorm1_init = -1.0; // Gnorm1_init is initialized at the first iteration
     std::vector<double> beta(l);
     std::vector<double> QD(l);
+    std::vector<double> lambda(l);
+    std::vector<double> upper_bound(l);
```

Comment:
Mentioned commit in scikit-learn also changes `#define GETI(i) (i)` and adds initialization of `lambda` and `upper_bound` arrays: https://github.com/scikit-learn/scikit-learn/blob/dcebbc4c5c97f886a57b0684a5e943f94db1bd58/sklearn/svm/src/liblinear/linear.cpp#L1073-L1089
And assuming we have L2R_L1LOSS_SVR_DUAL case here, single-element arrays make sense, but `GETI` macro is wrong.

Alternatively there is another source of original code where `GETI(i)` is `0` and arrays of size 1 works well: https://github.com/cjlin1/liblinear/blob/8dc206b782e07676dc0d00678bedd295ce85acf3/linear.cpp#L1127